static/js/task-feed/controllers/task-feed.controller.js

```diff
@@ -27,8 +27,44 @@
       self.modules = [];
 
       TaskFeed.getProjects().then(
-        function success (data) {
-            self.modules = data[0];
+        function success (successData) {
```

Comment:
this one does not work, take the current version which fixes this (work in progress on task feed)